src/core/animation-loop.js

```diff
@@ -258,4 +266,10 @@ export default class AnimationLoop {
       resizeDrawingBuffer(this.gl.canvas, {useDevicePixels: this.useDevicePixels});
     }
   }
+
+  _resizeFramebuffer() {
+    if (this.framebuffer) {
+      this.framebuffer.resize({width: this.gl.canvas.width, height: this.gl.canvas.height});
```

Comment:
Should we change this to `drawingBufferWidth` and `drawingBufferHeight` as well ?